### DISCUSSION TITLE
[mongodb] filtering using `where` not working.

### DISCUSSION THREAD

**Discussion by bn-piar** [47ng/prisma-field-encryption#40] [Q&A]
```prisma
model User {
  id           String   @id @default(auto()) @map("_id") @db.ObjectId
  email        String   @unique /// @encrypted
  emailHash    String?  @unique /// @encryption:hash(email)
  password     String /// @encrypted
  passwordHash String? /// @encryption:hash(password)
  name         String?
  role         UserRole @default(Client)
  createdAt    DateTime @default(now())
  updatedAt    DateTime @default(now()) @updatedAt
}

enum UserRole {
  Admin
  Client
  Sale
}
```
Hi. I added a document to db with field `{ email: "hehe" }`, however this does not work `const user = await prisma.user.findUnique({ where: { email: "hehe" } });`

Any clues?

**Comment by franky47** [comment]
Could you try enabling [debugging printouts](https://github.com/47ng/prisma-field-encryption#debugging), to see what comes up?

Edit: also, you should never store passwords in an accessible state (plain text or encrypted). See the [disclaimer](https://github.com/47ng/prisma-field-encryption#obligatory-disclaimer-about-passwords). One day I might make a Prisma middleware that does this the right way transparently.
